File: week_6/sentimental-mario-more/mario.py
# Loop to take input
while True:
    height = input("Height:")
    # Check weather input is a digit or not
    if (height.isdigit()):
        # Check if the input is between 0 and 8
        if (int(height) > 0 and int(height) <= 8):
            break
        else:
            continue
    else:
        continue
# Printing the pattern
height = int(height)
for i in range(1, height+1):
    print(" "*(height-i) + "#" * i, end='  ')
    print("#" * i)

File: week_1/pset1/mario-more/mario.c
#include<cs50.h>
#include <stdio.h>

int get_height();
int main(void)
{
    int h = get_height();
    int i, k ;
    //For loop to print the pattern
    for (i = 1 ; i <= h ; i++)
    {
        //For loop to print the spaces
        for (k = 0 ; k < h - i  ; k++)
        {
            printf(" ");
        }
        //For loop to print #
        for (k = 0 ; k < i ; k++)
        {
            printf("#");
        }
        //Print Statement to print the spaces of 2 between each of the line
        printf("  ");
        //Another loop to print after 2 spaces
        for (k = 0 ; k < i ; k++)
        {
            printf("#");
        }

        //Statement to move to next line
        printf("\n");

    }

}
//Function to take input
int get_height(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    return n;
}

